databridge_etl_tools/oracle/oracle_commands.py

```diff
@@ -23,3 +24,15 @@ def extract(ctx):
 def extract_json_schema(ctx): 
     """Extracts a dataset's schema in Oracle into a JSON file in S3"""
     ctx.obj.load_json_schema_to_s3()
+
+@oracle.command()
+@click.pass_context
+def truncate_and_load(ctx,): 
```

Comment:
Why do we need `truncate_and_load()` and what is it doing that is different from `load()`? Could we delete `truncate_and_load()`?

It does not seem like we have a simple append method though

Reply:
reworked as append